frontend/packages/ux-editor-v3/src/components/config/componentSpecificContent/Map/MapComponent.test.tsx

```diff
@@ -5,7 +5,7 @@ import { MapComponent } from './MapComponent';
 import { renderWithMockStore, renderHookWithMockStore } from '../../../../testing/mocks';
```

Comment:
Alias can be used on line 5 and 6 😊 
```suggestion
import { renderWithMockStore, renderHookWithMockStore } from '@studio/testing/mocks';
```

Reply:
Same here as [my previous comment](https://github.com/Altinn/altinn-studio/pull/12797#discussion_r1611766890)